requirements/base.txt

```diff
@@ -2,6 +2,7 @@ django==2.0.3
 django-environ==0.4.4
 boto3==1.6.7
 django-model-utils==3.1.1
+pika==0.11.2
```

Comment:
I see we pull `pika` in, but we don't actually use it anywhere. Remove it?

Reply:
Sure. It will come into play in the next story, and I'll use it to demo, but it doesn't need to be here yet.

Reply:
Mostly because I'm not familiar with the differences: why Pika instead of Celery? Or is it not an either/or thing?